src/definitions/modules/dropdown.js

```diff
@@ -110,8 +110,8 @@ $.fn.dropdown = function(parameters) {
               settings.ignoreDiacritics = false;
               module.error(error.noNormalize, element);
             }
-
-            module.setup.layout();
+            module.create.id();
+            module.setup.layout(id);
```

Comment:
The (id) parameter is not consumed inside the layout function.
As i suggested in https://github.com/fomantic/Fomantic-UI/pull/2409/files#r932673809 the "module.get.id()" function should be used instead, so please remove the provided id as a function call parameter.